utils/docker/Dockerfile.leap.15

```diff
@@ -86,71 +93,60 @@ RUN zypper --non-interactive update && \
     zypper clean --all
 
 # Monkey-patch MODULEPATH
-RUN if ! grep MODULEPATH=.*/usr/share/modules /etc/profile.d/lmod.sh; then        \
-        sed -e '/MODULEPATH=/s/$/:\/usr\/share\/modules/' /etc/profile.d/lmod.sh; \
+RUN if ! grep MODULEPATH=.*/usr/share/modules /etc/profile.d/lmod.sh; then \
+        sed -e '/MODULEPATH=/s/$/:\/usr\/share\/modules/'                  \
+               /etc/profile.d/lmod.sh; \
     fi
 
 RUN update-ca-certificates
-RUN zypper --non-interactive --no-gpg-checks install --allow-unsigned-rpm lua-lmod
-
-RUN if [ -z "${REPO_GROUP_LEAP15}" ]; then                                                                                        \
-        zypper --non-interactive addrepo                                                                                          \
-        https://download.opensuse.org/repositories/devel:languages:go/openSUSE_Leap_15.1/devel:languages:go.repo;                 \
-        zypper --non-interactive --gpg-auto-import-keys refresh;                                                                  \
-    else                                                                                                                          \
-         # Need the GPG key for the GO language repo (part of the group repo above)                                               \
-         rpm --import "${REPO_URL}${REPO_GROUP_LEAP15%/*}/opensuse-15.2-devel-languages-go-x86_64-proxy/repodata/repomd.xml.key"; \
-    fi
-# We actually only need go 1.12 or higher, but SUSE's 1.12 package is actually 1.11
-# https://bugzilla.opensuse.org/show_bug.cgi?id=1178985
-RUN zypper --non-interactive install -y ipmctl-devel go1.14 go1.14-race
+RUN zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks ref && \
+    zypper --non-interactive --no-gpg-checks install --allow-unsigned-rpm \
```

Comment:
GPG checking again. I think we are getting this from upstream now. But even if not, this should be set at the repo level, not per command.